extensions/lifecycle/tasks/LifecycleDeleteObjectTask.js

```diff
@@ -104,24 +104,12 @@ class LifecycleDeleteObjectTask extends BackbeatTask {
     }
 
     _getS3Action(actionType, accountId) {
-        let reqMethod;
         if (actionType === 'deleteObject') {
             const backbeatClient = this.getBackbeatClient(accountId);
-            if (!backbeatClient) {
-                return null;
-            }
-            // Zenko supports the "deleteObjectFromExpiration" API, which
-            // sets the proper originOp in the metadata to trigger a
-            // nortification when an object gets expired.
-            if (typeof backbeatClient.deleteObjectFromExpiration === 'function') {
-                return backbeatClient.deleteObjectFromExpiration.bind(backbeatClient);
-            }
-            reqMethod = 'deleteObject';
-        } else {
-            reqMethod = 'abortMultipartUpload';
+            return backbeatClient?.deleteObjectFromExpiration?.bind(backbeatClient);
```

Comment:
* this requires the new API being added to Cloudserver? If it is not present (e.g. cloudserver not bumped), what will happen? should we fallback to the regular `deleteObject` on "not implemented" error ?
* what happens if `backbeatClient?.deleteObjectFromExpiration? == null` : is this handled in the code? Either we should handle that (unlikely) scenario, or we should remove the conditionals and test that it does work indeed (e.g. unit test with `assert(_getS3Action('deleteObject) != null)`)